index.js

```diff
@@ -0,0 +1,31 @@
+export * from "./pkg/matrix_sdk_crypto_wasm_bg.js";
+import * as wasm from "./pkg/matrix_sdk_crypto_wasm_bg.js";
+
+const moduleUrl = new URL("./pkg/matrix_sdk_crypto_wasm_bg.wasm", import.meta.url);
```

Comment:
for reasons I don't entirely understand, webpack is putting the wasm in the top-level build directory, rather than in a `bundle` directory. That's probably a problem in element-web's webpack config, but it would be good to fix before we ship this.

Reply:
I also noticed that, no idea why. Element Call (so, Vite) properly handled this as regular asset.